http/README.md

```diff
@@ -28,14 +25,13 @@ This object MUST NOT contain any properties. Its name is reserved for future use
 <a name="operation"></a>
 
 ## Operation Binding Object
-
+ 
 ##### Fixed Fields
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="operationBindingObjectType"></a>`type` | string | **REQUIRED**. Type of operation. Its value MUST be either `request` or `response`.
-<a name="operationBindingObjectMethod"></a>`method` | string | When `type` is `request`, this is the HTTP method, otherwise it MUST be ignored. Its value MUST be one of `GET`, `POST`, `PUT`, `PATCH`, `DELETE`, `HEAD`, `OPTIONS`, `CONNECT`, and `TRACE`.
-<a name="operationBindingObjectQuery"></a>`query` | [Schema Object][schemaObject] | A Schema object containing the definitions for each query parameter. This schema MUST be of type `object` and have a `properties` key.
+<a name="operationBindingObjectMethod"></a>`method` | string | The HTTP method for the request. Its value MUST be one of `GET`, `POST`, `PUT`, `PATCH`, `DELETE`, `HEAD`, `OPTIONS`, `CONNECT`, and `TRACE`.
+<a name="operationBindingObjectQuery"></a>`query` | [Schema Object][schemaObject] | A Schema object containing the definitions for each query parameter. This schema MUST be of type object and have a properties key.
```

Comment:
Why not Multi Format one? We can allow schemas from OpenAPI (query parameters) for example.

Reply:
@smoya the OpenAPI Parameter Object has a `schema` property that is actually the only useful thing for us here. If someone wants to reuse, they can point to the `schema` property of the OpenAPI parameter.

Reply:
👍